changelog.d/20231025_101044_boris_keypoints.md

```diff
@@ -0,0 +1,4 @@
+### Fixed
+
+- A mask becomes visible even if hidden after changing opacity level
+  (<https://github.com/opencv/cvat/pull/7060>)
```

Comment:
Need to change the name of this file

Reply:
I do not think the file name is important. All these files will be removed after release